.gitignore

```diff
@@ -10,7 +10,6 @@ _build/
 *.war
 *.nar
 *.ear
-*.zip
```

Comment:
This should stay ignored. You don't need to add generated files to source control.

Reply:
Looks like you beat me by seconds.